### ISSUE TITLE
[BUG] Hover tooltip breaks with full-circle wedge

### ISSUE THREAD

**Issue by PavloFesenko** [bokeh/bokeh#9152]
For defects or deficiencies, please provide *ALL OF THE FOLLOWING*:

#### ALL software version info (bokeh, python, notebook, OS, browser, any other relevant packages)
bokeh 1.3.0, python 3.6.6, notebook Kaggle, OS Windows 10, browser Chrome

#### Description of expected behavior and the observed behavior
When all of the values (except one) in the wedge/pie chart are 0, the hover tooltip doesn't show any value.

#### Complete, minimal, self-contained example code that reproduces the issue

```
from math import pi

import pandas as pd

from bokeh.io import output_file, show
from bokeh.palettes import Category20c
from bokeh.plotting import figure
from bokeh.transform import cumsum

output_file("pie.html")

x = {
    'United States': 157,
    'United Kingdom': 0,
    'Japan': 0,
    'China': 0,
    'Germany': 0,
    'India': 0,
    'Italy': 0,
    'Australia': 0,
    'Brazil': 0,
    'France': 0,
    'Taiwan': 0,
    'Spain': 0
}

data = pd.Series(x).reset_index(name='value').rename(columns={'index':'country'})
data['angle'] = data['value']/data['value'].sum() * 2*pi
data['color'] = Category20c[len(x)]

p = figure(plot_height=350, title="Pie Chart", toolbar_location=None,
           tools="hover", tooltips="@country: @value", x_range=(-0.5, 1.0))

p.wedge(x=0, y=1, radius=0.4,
        start_angle=cumsum('angle', include_zero=True), end_angle=cumsum('angle'),
        line_color="white", fill_color='color', legend='country', source=data)

p.axis.axis_label=None
p.axis.visible=False
p.grid.grid_line_color = None

show(p)

```

#### Screenshots or screencasts of the bug in action

The cursor isn't visible but it is on the pie chart.

![Bokeh bug](https://user-images.githubusercontent.com/16855796/62786906-d9faeb80-bac3-11e9-9765-3a6a9b5c8386.png)

**Comment by mattpap** [comment]
The problem is with `angle_between()` in `core/util/math`, which equates `2*PI` angle distance to zero.

**Comment by bryevdv** [comment]
Hrm that's strange because `angle_norm` has strict comparisions, it should leave 2*pi alone:

https://github.com/bokeh/bokeh/blob/d213e3d755f09f57d2a58da3c1dcba939a3b90bf/bokehjs/src/lib/core/util/math.ts#L3-L11

**Comment by pirovc** [comment]
@bryevdv This bug was indeed fixed on bokeh version 1.4.0 but is re-occurring in recent versions (tested on 2.2.3 and 2.3.1). Should I open a new BUG report?

**Comment by bryevdv** [comment]
@pirovc It's been a long time and the codebase has changed considerably, please open a new issue.
